src/Common/EmptyState/emptyState.scss

```diff
@@ -21,17 +21,19 @@
     }
     .subtitle {
         font-size: 13px;
-        width: 300px;
```

Comment:
It can break existing component in some case. Have you verified those?

Reply:
Have added 300px as max width on the container